### ISSUE TITLE
Gear emoji is seen as misspelt

### ISSUE THREAD

**Issue by nicholaswilde** [tbroadley/spellchecker-cli#87]
When using a [gear emoji](https://emojipedia.org/gear/) in a markdown file, `spellchecker` flags the emoji but should probably be skipped.

### Steps to reproduce the issue
```shell
echo ":gear:" > test.md 
spellchecker test.md
Spellchecking 1 file...

test.md
  1:2-1:3  warning  `️` is misspelt; did you mean `a`, `b`, `c`, `d`, `e`, `f`, `g`, `h`, `i`, `j`, `k`, `l`, `m`, `n`, `o`, `p`, `q`, `r`, `s`, `t`, `u`, `v`, `w`, `x`, `y`, `z`?  retext-spell  retext-spell

⚠ 1 warning
```

### Expected behavior
```shell
echo ":gear:" > test.md 
spellchecker test.md 
Spellchecking 1 file...

test.md: no issues found
```

spellchecker version: `v4.11.0`

```shell
uname -a
Linux amd 5.15.0-33-generic #34-Ubuntu SMP Wed May 18 13:34:26 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by tbroadley** [comment]
Good catch. This might be as simple as upgrading the version of the remark-gemoji-to-emoji package that this project uses.

**Comment by tbroadley** [comment]
It looks like remark-gemoji-to-emoji hasn't been upgraded in a while. Seems like there's an equivalent package called remark-gemoji that's under @remark, so that's promising.

I'm having difficulty using this package, though. I get type errors that I'm having trouble understanding. I think they might be related to using an old version of Remark. But I'm also stuck on upgrading Remark because Remark v14 moved from CommonJS modules to ES modules and I haven't yet figured out how to use the latter in this project.

**Comment by nicholaswilde** [comment]
Bah, it's the worst when relying on libraries that are out of date. I feel your pain. Thanks for looking into this so quickly 🙏🏽. 

Do you happen to know how to bypass it for now by adding an entry to `dictionary.txt`?

**Comment by nicholaswilde** [comment]
Using a literal ⚙️ worked, although not ideal. Thanks for the help.
